app/helpers/logger/external-logger-bigquery.js

```diff
@@ -28,7 +28,9 @@ const ExternalLoggerBigQuery = function (bqClient, displayId, cacheVersion, os,
       display_id: displayId || "no-displayId",
       cache_version: cacheVersion || "no-risecache",
       os: os,
-      ts: nowDate.toISOString()
+      ts: nowDate.toISOString(),
+      file_name: fileName || fileSystem.getFileName(fileUrl),
```

Comment:
I think we can use default parameters  here - https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters